dotCMS/src/main/java/com/dotcms/publisher/pusher/DependencySet.java

```diff
@@ -0,0 +1,166 @@
+package com.dotcms.publisher.pusher;
+
+import com.dotcms.publisher.util.PusheableAsset;
+import com.dotcms.publishing.PublisherConfig.Config;
+import com.dotmarketing.beans.Host;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Use by {@link com.dotcms.publisher.pusher.PushPublisherConfig} to keep track of all the
+ * assets added into a Bundle.
+ */
+class DependencySet {
```

Comment:
Rename to BundleAssets

Reply:
done https://github.com/dotCMS/core/pull/20521/commits/078b38ddfdf91f36ed147df16700c696a0da57c1#diff-4620da1e420b91c427e5941f3851da9b7ea5ee39096809650bd047558ef5b2d0L13